### ISSUE TITLE
Application causes stuttering and CPU spikes with the latest version

### ISSUE THREAD

**Issue by zeknoss** [SibrenVasse/deezer#11]
Hello Sibren,
With the latest version I've started observing stuttering and minor freezes on my computer. Would you know something about it?
The deezer web app does not cause this issue.

**Comment by SibrenVasse** [comment]
I'm not encountering any issues here. Do they persist if you downgrade to 4.31?

**Comment by zeknoss** [comment]
Hey @SibrenVasse,
After downgrade, the problem went away again.

**Comment by SibrenVasse** [comment]
I cannot seem to reproduce the issue on my machine. What version of electron are you using? Are you using arch linux?

**Comment by zeknoss** [comment]
@SibrenVasse sorry for the lack of details I've provided.
My setup is as follows:
Manjaro Kernel v5.10.7-3
Electron v11.2.0
Also I've a very powerful setup with a Ryzen 5600x.

**Comment by SibrenVasse** [comment]
Deezer does not use electron v11, but the package electron6.
What version do you have installed?

**Comment by zeknoss** [comment]
I'm not sure I caught the last one. Which package's version are you referring to?

**Comment by SibrenVasse** [comment]
Deezer is started with this command (uses the package [electron6](https://archlinux.org/packages/community/x86_64/electron6/)):
```
exec electron6 /usr/share/deezer/app.asar "$@"
```
So the electron6 version is relevant.

**Comment by zeknoss** [comment]
On my computer it's started with the following command: `/usr/bin/deezer %u` and no version is involved in the application launch.

**Comment by SibrenVasse** [comment]
Please look inside the /usr/bin/Deezer script

**Comment by zeknoss** [comment]
It runs like yours does:
`exec electron6 /usr/share/deezer/app.asar "$@"`

**Comment by SibrenVasse** [comment]
Yes. So what is your electron6 version?

**Comment by zeknoss** [comment]
It is `v6.1.1.12`

**Comment by SibrenVasse** [comment]
So apart from you using Manjaro the environment seems to be the same. I cannot reproduce your issue.
Can you run `deezer` from the command line and submit the log. Please remove any personal information (mostly the 'sid' cookie)

**Comment by SibrenVasse** [comment]
Please also try v4.32.10

**Comment by zeknoss** [comment]
```
18:39:33.984 › Init App
18:39:34.011 (main) › App is ready
18:39:34.019 (main) › I18n: The ressources for the language "en" have been loaded.
xdg-mime: application argument missing
Try 'xdg-mime --help' for more information.
18:39:34.440 (main) › Creating Window
18:39:34.461 (main) › I18n: The language "en" has been saved into the "dz_lang" cookie.
18:39:34.470 (main) › Network: the status is now online.
18:39:34.470 (main) › Cookie: Persisting the cookie "dz_lang" with value "en" for two weeks.
18:39:34.476 (main) › Cookie: The cookie store has been flushed successfully.
18:39:35.492 (main) › Cookie: Persisting the cookie "sid" with value "fr5941c2c435b43c5992a281d5682d54a2ac8724" for two weeks.
```
This is all I got :) Nothing too informative I imagine.
I will inform you regarding the newer version in an hour or so.

**Comment by zeknoss** [comment]
Hey @SibrenVasse ,
I'll tell you something weird. I've been trying to upgrade Deezer back to the latest version for the last 2 days but there was some issues in the system and I finally found time to fix them. Right now I've upgraded to the latest version and the stutters are all gone.
Sorry for all the trouble and I deeply appreciate the time and effort you put into this weird issue. Thank you very much!
